### PR TITLE
Fix (SidebarLink, ScrollBar): bug fixes

### LINKED ISSUE #412: 1.9.0 bugs
Check these are obsolete and components are implemented.

- [ ] Select too long https://user-images.githubusercontent.com/5394573/49163283-c8aaee00-f33d-11e8-8242-9d5a4c1e637b.png

### PR DESCRIPTION
**Pull request by vlad-shusterman**
close #412

### CODE REVIEW

File: src/vuestic-theme/vuestic-components/vuestic-scrollbar/VuesticScrollbar.vue [hunk 1]
```diff
@@ -26,8 +38,34 @@ export default {
...
+    setItemsHeight (count) {
+      if (this.itemsCount !== null) {
+        return `items-height-${count}`
+      }
+    },
+    checkCoordinate (e) {
+      this.previousCoordinate = e.clientY
+    },
+    thumbScroll (e) {
+      if (this.isScrolling) {
```
[the remark this answers is not in the document]
Comment (asvae) [reply src/vuestic-theme/vuestic-components/vuestic-scrollbar/VuesticScrollbar.vue]: @jean-moldovan no place for disagreement here. : 3

File: src/vuestic-theme/vuestic-sass/resources/_variables.scss [hunk 2]
```diff
@@ -434,3 +434,12 @@ $toast-icon-size: $popover-icon-size;
 $toast-icon-color: $popover-icon-color;
 $toast-icon-ml: 0;
 $toast-icon-mr: 0.625rem;
+
+// Dropdown Values for simple select
+
+$simple-select-items: (
```
Comment (asvae) [inline src/vuestic-theme/vuestic-sass/resources/_variables.scss]: What are you even doing  here?

File: src/vuestic-theme/vuestic-components/vuestic-scrollbar/VuesticScrollbar.vue [hunk 3]
```diff
@@ -166,15 +206,21 @@ export default {
     position: relative;
     overflow: hidden;
     max-height: 100%;
+    @each $count, $height in $simple-select-items {
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-scrollbar/VuesticScrollbar.vue]: Remove that and just pass height directly to select.

File: src/vuestic-theme/vuestic-components/vuestic-scrollbar/VuesticScrollbar.vue [hunk 4]
```diff
@@ -9,8 +15,14 @@
            @transitionend="onContentResize">
         <slot></slot>
       </div>
-      <div class="track" ref="track">
-        <div class="thumb" ref="thumb"></div>
+      <div class="track" ref="track"
+           @mousemove="thumbScroll"
+      >
+        <div class="thumb" ref="thumb"
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-scrollbar/VuesticScrollbar.vue]: There is no need for scrollbar here. It's just visual.

File: src/components/admin/app-sidebar/components/SidebarLink.vue [hunk 5]
```diff
@@ -47,6 +47,7 @@ export default {
     height: $sidebar-link-height;
     padding-left: $sidebar-link-pl;
     display: flex;
+    flex-wrap: wrap;
```
Comment (asvae) [inline src/components/admin/app-sidebar/components/SidebarLink.vue]: What is that for?

File: src/components/admin/app-sidebar/components/SidebarLinkGroup.vue [hunk 6]
```diff
@@ -85,7 +85,10 @@ export default {
     }
     .sidebar-link__router-link {
       height: $sidebar-submenu-link-height;
-      padding-left: $sidebar-submenu-link-pl;
+      display: flex;
```
Comment (asvae) [inline src/components/admin/app-sidebar/components/SidebarLinkGroup.vue]: Compare visuals with master.

**Comment by smartapant** [comment]
@asva can we close this one?

**Comment by asvae** [comment]
Likely outdated.

**Pull request closed without merging**